frontend/src/app/(navfooter)/credits/page.tsx

```diff
@@ -26,6 +26,7 @@ const ICON_SOURCES = {
     "Tiger-like-x by Althepal": "https://commons.wikimedia.org/wiki/File:Tiger-like-x.svg",
     "Saturn by JustDanPatrick": "https://upload.wikimedia.org/wikipedia/commons/archive/7/78/20220518145749%21Sega_Saturn_Black_Logo.svg",
     "MS-DOS by Microsoft": "https://commons.wikimedia.org/wiki/File:Msdos-icon.svg",
+    "Tux by Larry Ewing": "https://commons.wikimedia.org/wiki/File:Tux.svg",
```

Comment:
it appears there are 3 authors actually - probably best to list all of them. (Simon Budig and Garrett LeSage too)